src/main/java/task/TaskList.java

```diff
@@ -0,0 +1,81 @@
+package task;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * TaskList class are used to store all the task specified
+ * by user in an ArrayList of Task.
+ */
+public class TaskList {
+
+    private List<Task> tasks;
+
+    /**
+     * Constructs a new TaskList with the tasks
+     * initalize to a new ArrayList
+     */
+    public TaskList() {
+        tasks = new ArrayList<Task>();
+    }
+
+    /**
+     * Constructs a new TaskList with the
+     * specified list of tasks.
+     * @param tasks List(Task) List of task
```

Comment:
Will it be better if there can be a new line between the description of the method and the description of the parameter? I have also noticed similar problems in several other places.